base/sparse/sparsematrix.jl

```diff
@@ -2936,11 +2920,11 @@ function blkdiag(X::SparseMatrixCSC...)
 end
 
 ## Structure query functions
-issymmetric(A::SparseMatrixCSC) = is_hermsym(A, IdFun())
+issymmetric(A::SparseMatrixCSC) = is_hermsym(A, identity)
 
-ishermitian(A::SparseMatrixCSC) = is_hermsym(A, ConjFun())
+ishermitian(A::SparseMatrixCSC) = is_hermsym(A, conj)
 
-function is_hermsym(A::SparseMatrixCSC, check::Func)
+function is_hermsym(A::SparseMatrixCSC, check::Function)
```

Comment:
@vtjnash Did the following comment on my PR that changed the same thing as this one: https://github.com/JuliaLang/julia/pull/15696#discussion_r58486970